packages/matplotlib/src/html5_canvas_backend.py

```diff
@@ -55,14 +52,15 @@ def download(self, format, mimetype):
         # content, and then virtually clicks it. Kind of magical, but it
         # works...
         element = document.createElement('a')
-        data = io.BytesIO()
-        try:
-            self.canvas.figure.savefig(data, format=format)
-        except Exception:
-            raise
-        element.setAttribute('href', 'data:{};base64,{}'.format(
-            mimetype, base64.b64encode(data.getvalue()).decode('ascii')))
+
+        canvas = self.canvas.get_element('canvas')
+        imgURL = canvas.toDataURL('image/png')
```

Comment:
We should investigate why this isn't triggering a flake8 violation on CircleCI.

Reply:
What exactly? the variable name?  I don't find any violation here :/

Reply:
Interesting -- that variable name is a PEP8 violation, but maybe flake8 can't fix it itself.  It should be `img_url`.